influxdb_connector/influxdb_connector.py

```diff
@@ -89,37 +137,38 @@ def custom_bucket_formatting(csv_builder, bucket):
         A dictionary containing time and field columns from influxdb query
     """
     # custom formatting for anomaly detection and time series bucket
-    if bucket == 'anomaly_detection' or 'anomaly' in csv_builder:
-        csv_builder['anomaly'] = [1 if x == 3 else x for x in csv_builder['anomaly']]
-    elif bucket == 'ts_forecast':
+    if bucket == "anomaly_detection" or "anomaly" in csv_builder:
+        csv_builder["anomaly"] = [1 if x == 3 else x for x in csv_builder["anomaly"]]
+    elif bucket == "ts_forecast":
         pass
     else:
         raise InvalidBucketError(bucket)
-    
     return csv_builder
 
+
 def main():
     args = parse_parameters()
-    if args.token.lower() == 'secret':
-        args.token = os.environ.get('INFLUXDB_TOKEN')
-    if args.url.lower() == 'secret':
-        args.url = os.environ.get('INFLUXDB_URL')
-    if args.org.lower() == 'secret':
-        args.org = os.environ.get('INFLUXDB_ORG')
-    if args.bucket.lower() == 'secret':
-        args.bucket = os.environ.get('INFLUXDB_BUCKET')
-
+    if args.token.lower() == "secret":
+        args.token = os.environ.get("INFLUXDB_TOKEN")
+    if args.url.lower() == "secret":
+        args.url = os.environ.get("INFLUXDB_URL")
+    if args.org.lower() == "secret":
+        args.org = os.environ.get("INFLUXDB_ORG")
+    if args.bucket.lower() == "secret":
+        args.bucket = os.environ.get("INFLUXDB_BUCKET")
     # return dictionary from custom query
-    csv_builder = influxdb_query(args.url, args.token, args.org, args.bucket, args.range_start)
+    csv_builder = influxdb_query(
+        args.url, args.token, args.org, args.bucket, args.range_start
+    )
     csv_builder = custom_bucket_formatting(csv_builder, args.bucket)
 
     # build pandas dataframe for csv
     df = pd.DataFrame(csv_builder)
     df.dropna(inplace=True)
-    df.to_csv(args.local_dir+'/'+args.file_name, index=False)
+    df.to_csv(args.local_dir + "/" + args.file_name, index=False)
 
     # Store influxdb csv as cnvrg dataset
-    if args.cnvrg_dataset.lower() != 'none':
+    if args.cnvrg_dataset.lower() != "none":
```

Comment:
in the try except block below line 176 Try to catch a specific exception.  Flake8 will catch this